src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheckTest.java

```diff
@@ -141,30 +141,39 @@ public void testScopes() throws Exception {
             "51:9: " + getCheckMessage(MSG_JAVADOC_MISSING),
             "52:9: " + getCheckMessage(MSG_JAVADOC_MISSING),
             "53:9: " + getCheckMessage(MSG_JAVADOC_MISSING),
-            "63:5: " + getCheckMessage(MSG_JAVADOC_MISSING),
-            "64:5: " + getCheckMessage(MSG_JAVADOC_MISSING),
-            "65:5: " + getCheckMessage(MSG_JAVADOC_MISSING),
-            "66:5: " + getCheckMessage(MSG_JAVADOC_MISSING),
+        };
+        verifyWithInlineConfigParser(
+                getPath("InputJavadocVariableNoJavadocOne.java"),
+               expected);
+    }
+
+    @Test
+    public void testScopesTwo() throws Exception {
```

Comment:
Same as above.

Reply:
Done 👍🏻

Reply:
Not done, please make sure that you’ve pushed your changes up in GitHub.

Reply:
I have pushed the changes..

Reply:
Re: https://github.com/checkstyle/checkstyle/pull/14369#discussion_r1469097071

To be clear: naming conventions like BlahBlahOne in both test method names and test inputs tell us nothing about the purpose of a given test. Imagine that you  come back to this test class in a few months. Why does this test exist? What did the author want to validate? 

Do you think that an input file name such as `InputJavadocVariablePublicInnerClass` in a test method called `testJavadocVariablePublicInnerClass` would give you a better idea of what we are testing, without even needing to open up the input file and inspect the configuration and code?

Reply:
Also, I have made the changes requested on this [PR #14369 ](https://github.com/checkstyle/checkstyle/pull/14369).
Hope this is fine!!

Reply:
>Well in that case, won't we have to manually rename the input files and the test methods according to the checks performed?

I am not sure exactly what you are asking here. I will place exact suggestions on the inputs and test methods to make my request more clear.

Reply:
> > Well in that case, won't we have to manually rename the input files and the test methods according to the checks performed?
> 
> I am not sure exactly what you are asking here. I will place exact suggestions on the inputs and test methods to make my request more clear.

I was just curious on how do you propose to modify the files that are no longer a part of the issue #11163.
I have changed the naming as part of the current PR according to your suggestions, but what about the files that have already been merged previously?

Reply:
>but what about the files that have already been merged previously?

Ah, I understand now. "Fixing" these would require more effort than it is probably worth at this point. If we can just do better from here on out, this will still be a major improvement and beneficial for the next contributor.

After we merge this, we can update the issue description to use this PR as an example going forward to inspire others :)

Reply:
> One more thing, could it be possible to leave an Open Issue

> why not let the passionate contributors do the work - the whole point of open source....

Be the change you want to see, anyone is welcome to open issues. Approval is a different story, we will cross that bridge when we encounter it.